### DISCUSSION TITLE
Vaillant Ebus decoding help needed

### DISCUSSION THREAD

**Discussion by rollercontainer** [arendst/Tasmota#16681] [Q&A]
I've managed to receive and decode some values. I found lead, return and outside temp from B5h 11h (EcoTec Plus, VRC470). 
The manual (vaillant_ebus_v0.6.0.pdf) describes lead target and service target temps on byte 8 and 9 in the B5h 10h (09h) message. But I only get hard values 00 and ff. In the dump there is a second  B5h 10h with 03h data length but that isn't documented. 
```
18:06:04.134 : aa 10 08 b5 10 09 00 00 00 ff ff ff 01 00 00 26 00 01 01 9a 00 
18:06:02.134 : aa 10 08 b5 10 03 05 ff 01 98 00 01 01 9a 00 
```

Could anyone throw me a bone?
```

3,xxxxB51009xxxxuu@1,Vorlauf soll,C,lead target,0
3,xxxxB51009xxxxxxuu@1,Warmwasser soll,C,service target,0
3,xxxxB5110101xxxxxxuu@2,Vorlauf,C,lead,1
3,xxxxB5110101xxxxxxxxuu@2,Rücklauf,C,return,1
3,xxxxB5110101xxxxxxxxxxxxss@1,Aussen,C,outside,0
3,xxxxB5110101xxxxxxxxxxxxxxxxxxuu@1,Speicher,C,Speicher,0
3,xxxxB5110101xxxxxxxxxxxxxxxxxxxxuu@b0:1,Brenner,,heating,0
3,xxxxB5110101xxxxxxxxxxxxxxxxxxxxuu@b1:1,Speicherladung,,service water,0 
```

**Comment by Silverstar** [comment]
Are you aware of the ebusd project? There is a repository of configuration files for many devices, as well for the 470 (which I have as well).

https://github.com/john30/ebusd-configuration/blob/master/ebusd-2.1.x/de/vaillant/15.470.csv
